README.md

```diff
@@ -6,6 +6,17 @@ The output is presented in 2 format, json format(saved to running directory) and
 **Console OutPut**
 
 ```
+Solution-Level Data:
+-- Projects: 15
+-- WebFormsViewFiles: 24
+-- WebFormsCodeBehindFiles: 24
+-- UserControlFiles: 0
+-- UserControlCodeBehindFiles: 0
+-- MasterFiles: 0
```

Comment:
run a project with master files and user control files... all numbers are 0... e:g: - https://github.com/tomasherceg/ModernizeWebFormsSample

Reply:
The recommended project was better- it identified master files but did not have user controls. To test user controls, I tried the eShop Web Forms project: https://github.com/dotnet-architecture/eShopOnBlazor/tree/master/src/eShopLegacyWebForms